docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,293 @@
+    field to determine whether a block can be viewed on a device. This URL can
+    be used as a fallback if the ``student_view_data`` for this block type is
+    not supported by the client or by the block.
+
+  * lms_web_url: (string) The URL to the navigational container of the XBlock
+    on the web LMS. This URL can be used as a fallback if the
+    ``student_view_data`` and ``student_view_url`` fields are not supported.
+
+  * lti_url: (string) The block URL for an LTI consumer. Returned only if the
+    ``ENABLE_LTI_PROVIDER`` Django setting is set to ``True``.
+
+
+============================================================================
+Example Response Showing a List of Course Blocks in a Specified Course
+============================================================================
+
+The following example response is returned from this request:
+
+GET /api/courses/v1/blocks/?course_id=edX/DemoX/Demo_Course&all_blocks=true
+&block_counts=video,html,problem&requested_fields=graded,format,student_view_data,
+student_view_url,student_view_multi_device&student_view_data=video,html,problem
+
+
+.. code-block:: json
+
+ {
+    "root": "i4x://edX/DemoX/course/Demo_Course",
+    "blocks": {
+       "i4x://edX/DemoX/course/Demo_Course": {
+          "display_name": "edX Demonstration Course",
+          "graded": false,
+          "student_view_url": "https://courses.edx.org/xblock/i4x://edX/DemoX/
+           course/Demo_Course",
+          "student_view_multi_device": false,
+          "lms_web_url": "https://courses.edx.org/courses/edX/DemoX/Demo_Course/
+           jump_to/i4x://edX/DemoX/ course/Demo_Course",
+          "type": "course",
+          "id": "i4x://edX/DemoX/course/Demo_Course",
+          "block_counts": {
+             "problem": 23,
+             "html": 32,
+             "video": 5
+            }
+        }
+    }
+ }
+
+
+.. _Get a list of the course blocks in a block tree:
+
+*********************************************
+Get a List of Course Blocks in a Block Tree
+*********************************************
+
+The endpoint to get a list of course blocks in a specified block tree is
+``/api/courses/v1/blocks/{usage_id}/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of course blocks in a specified block tree. Response results depend
+on the specified user's permissions level within a course, as well as group
+membership and individual allowances (such as due date extensions), if any.
+
+=====================
+Request Format
+=====================
+
+GET /api/courses/v1/blocks/{usage_id}/
+
+Example:
+
+GET /api/courses/v1/blocks/i4x%3A%2F%2FedX%2FDemoX%2Fvertical
+%2F2152d4a4aadc4cb0af5256394a3d1fc7?all_blocks=true
+
+
+=====================
+Query Parameters
+=====================
+
+:ref:`Query parameters<Blocks Query Parameters>` for this endpoint are the
+same as those for :ref:`Get a list of the course blocks in a course`, with the
+addition of ``usage_id``.
```

Comment:
So, the `usage_id` is not a query parameter, technically.  Rather, it's part of the URL and specifies the resource that is being accessed.